app/views/unauthorised.scala.html

```diff
@@ -17,44 +17,59 @@
 @import config.ExternalServicesConfig
 @import views.helpers.Title
 @import views.html.components.gds._
+@import config.featureFlags.TdrUnauthorisedMsgConfig
 
 @this(
     govukLayout: gdsMainTemplate,
     pageTitle: pageTitle,
     paragraphBody: paragraphBody,
     link: link,
     bulletList: bulletList,
-    config: ExternalServicesConfig
+    config: ExternalServicesConfig,
+    tdrUnauthorisedMsgConfig: TdrUnauthorisedMsgConfig
 )
 
 @()(implicit request: Request[_], messages: Messages)
 
-@howToSection = {
-    @paragraphBody(messages("unauthorised.paragraph.1"))
-
-    @bulletList(List(
-      Html(messages("unauthorised.paragraph.1.bullet.1", link(
-        id = Some("get_eori_link"),
-        text = messages("unauthorised.paragraph.1.bullet.1.link"),
-        call = Call("GET", config.eoriService)
-      ))),
-      link(
-        id = Some("access_cds_link"),
-        text = messages("unauthorised.paragraph.1.bullet.2.link"),
-        call = Call("GET", config.cdsRegister)
-      )
-    ))
-}
+@if(tdrUnauthorisedMsgConfig.isTdrUnauthorisedMessageEnabled) {
+    @govukLayout(title = Title("unauthorised.tdr.heading")) {
+        @pageTitle(messages("unauthorised.tdr.heading"))
 
-@govukLayout(title = Title("unauthorised.heading")) {
+        @paragraphBody(messages("unauthorised.tdr.body",
+            link(
+                id = Some("contact_support_link"),
+                text = messages("unauthorised.tdr.body.link"),
+                call = Call("GET", s"mailto:${messages("unauthorised.tdr.body.link")}"),
+                target="_blank"
+            ))
+        )
+    }
+} else {
+    @govukLayout(title = Title("unauthorised.heading")) {
+        @pageTitle(messages("unauthorised.heading"))
 
-    @pageTitle(messages("unauthorised.heading"))
+        @paragraphBody(messages("unauthorised.paragraph.1"))
 
-    @howToSection
+        @bulletList(List(
```

Comment:
Could not the `bulletList` be extracted, placed under flag condition and assigned to a "var", so to have a single `govukLayout` that includes the "maybe bulletList var"?

This seem more DRY to me.

Reply:
But the two `govukLayout` instances are passed two very different title values and content blocks. The bullet list is not the only difference between the two conditional branches so not sure what could be gained :thinking:

Reply:
You're right :+1:  I did not notice the other differences between the 2 layouts.